File: app/src/main/java/br/com/lumean/enews/models/Team.java
package br.com.lumean.enews.models;

public class Team {
    private String name;
    private String imageUrl;
    private String twitterUrl;

    private Team() {}

    public Team(String nome, String imageUrl, String twitterUrl) {
        this.name = nome;
        this.imageUrl = imageUrl;
        this.twitterUrl = twitterUrl;
    }

    public String getName() {
        return name;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public String getTwitterUrl() {
        return twitterUrl;
    }
}

File: app/src/main/java/br/com/lumean/enews/models/Team.kt
package br.com.lumean.enews.models;

import java.io.Serializable


data class Team (
    val name : String = "",
    val imageUrl : String = "",
    val twitterUrl : String = "",
    val backgroundUrl : String = "",
    val players : List<Player> = emptyList()
) : Serializable